### DISCUSSION TITLE
Client for Android

### DISCUSSION THREAD

**Discussion by edutsilva** [GerryDazoo/Slinger#180] [Ideas]
First of all, I would like to thank Gerry for this solution. Amazing. Easy to setup and works beautifully.

I´m no coder, but it would be awasome if a slingbox owner that can code could make an android app to connect to our stand-alone servers. 
Just an idea.
Cheers!

**Comment by GerryDazoo** [comment]
??? Not sure what you mean. There are a number of Android media player apps that are compatible. VLC, MXPlayer....

**Reply by edutsilva** [reply]
Hi, i know that. What I mean is an app designed to connect to our stand along slingbox servers. Like the slingbox app itself. With a remote control inside. Just a suggestion.
Thank you again.
